components/generic/ContentCard.vue

```diff
@@ -52,6 +52,14 @@
         >
           {{ $d(new Date(datetime), 'short') }}
         </time>
+        <template v-if="hitsText">
+          <b-card-text
+            text-tag="div"
+            data-qa="highlighted search term"
+          >
+            <p>{{ hitsText.prefix }} <span class="font-weight-bold has-text-highlight">{{ hitsText.exact }}</span> {{ hitsText.suffix }}</p>
```

Comment:
Let's use strong instead of span here, for better semantic markup and accessibility.